react/src/components/ImportFromHuggingFaceModal.tsx

```diff
@@ -132,7 +158,41 @@ const ImportFromHuggingFaceModal: React.FC<ImportFromHuggingFaceModalProps> = ({
     formRef.current
       ?.validateFields()
       .then((values) => {
-        // TODO: Implement import from Hugging Face
+        importAndStartService.mutate(
+          {
+            huggingFaceUrl: values.url,
+            importOnly: isImportOnly,
+            serviceName: values.service_name || undefined,
+            folderName: values.folder_name || undefined,
+          },
+          {
+            onSuccess(data: any) {
+              if (data?.folder?.id && data?.folder?.name) {
+                upsertNotification({
+                  key: 'ImportFromHuggingFaceModal' + data.folder.id,
+                  message: `${t('data.modelStore.NewModelStoreFolderHasBeenCreated')}: ${data.folder.name}`,
```

Comment:
You can use `description` too.

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/XqC2uNFuj0wg8I60sMUh/c7e631a7-70d9-4c01-943c-86101ca5ad7f.png)

```suggestion
                  message: `${t('data.modelStore.NewModelStoreFolderHasBeenCreated')}`,
                  description: data.folder.name,
```